### ISSUE TITLE
Add Custom Design Back Product

### ISSUE THREAD

**Issue by mtuttle03** [AMTuttle02/HomeTeamCreativity#213]
I want to add a custom design product for back shirts, like we have one for fronts, I can upload the design and everything, but I would need it to be a range of price and add the option for "upload image below", and require custom box. Pretty much everything for custom design on there, just duplicated for back shirts. Also, would want this to be at the very top of the products page please. I didn't want to upload it myself if you want me to do something different. Let me know, thanks!!!

**Comment by AMTuttle02** [comment]
@mtuttle03 I think instead, we should just add a back design file to the existing custom design product. That way there is one custom design product but it showcases that you can do front or back. Thoughts on that?

**Comment by mtuttle03** [comment]
@AMTuttle02 yes that is what I wanted, use same design as front, just show that custom design can be on front or back. thank yew

**Comment by mtuttle03** [comment]
looks good!
